### PR TITLE
Edit headings

### PR DESCRIPTION
**Pull request by Alkawi**
Make headings bigger & bold

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,32 +4,36 @@
...
       <img src="images/alexander.jpg" alt="Alexander Kaiser" />
-      <h2>About me</h2>
+      <h2 id="aboutme" class="text-2xl font-bold">About me</h2>
       <p>
         My name is Alexander Kaiser. I am 24 yours old and live in Wippingen,
         Lower Saxony. At the moment I am participating in a web development
         bootcamp.
       </p>
-      <h3>Hobbies</h3>
+      <h3 class="text-xl font-bold">Hobbies</h3>
```
Comment (PReichetanz) [inline index.html:29]: You consider "Hobbies" as a subsection of "About me", right? Otherwise, it should be an h2 as well.

**Comment by PReichetanz** [comment]
I like it!

File: index.html (hunk 1 above)
Comment (Alkawi) [reply index.html:29]: Correct, it belongs to the "About me" section

File: index.html [hunk 2]
```diff
@@ -4,32 +4,36 @@
...
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1 id="top">Alexander Kaiser</h1>
+  <body class="m-4">
+    <h1 id="top" class="text-3xl font-bold mb-4">Alexander Kaiser</h1>
     <nav>
-      <ul>
+      <ul role="list">
```
Comment (badabam) [inline index.html:16]: Yeah! Nice!

**Review by badabam — COMMENTED** [review]
Nice work!

**Pull request merged**